File: eng-debugging/hello_world_java/src/test/java/HelloWorldTest.java
import org.junit.Test;

import java.io.IOException;
import java.util.Arrays;
import java.util.List;

import static org.junit.Assert.assertEquals;

public class HelloWorldTest {
    @Test
    public void testCase1() throws IOException {
        // This test case is passing
        HelloWorld helloWorld = new HelloWorld("input/test_1.txt");
        String actualOutput = helloWorld.run();
        assertEquals("hello world\n", actualOutput);
    }

    @Test
    public void testCase2() throws IOException {
        // This test case is failing
        HelloWorld helloWorld = new HelloWorld("input/test_2.txt");
        String actualOutput = helloWorld.run();
        assertEquals("hello-world\n", actualOutput);
    }
}

File: eng-debugging/hello_world_python/test/hello_world_test.py
import pytest
import os
from lib.index import HelloWorld
input_folder = "input/"

def test_case_one():
    hello_world = HelloWorld(input_folder + "test_1.txt")
    assert hello_world.run() == "hello world\n"


def test_case_two():
    hello_world = HelloWorld(input_folder + "test_2.txt")
    assert hello_world.run() == "hello-world\n"
